helm/clickhouse/values.yaml

```diff
@@ -298,4 +298,5 @@ pluralRunbook:
   enabled: true
 
 pre-delete-installation:
+  image: bitnami/kubectl:1.25.8
```

Comment:
Please use the default formatting so Renovate picks it up properly and we can keep this updated automatically:

```yaml
image:
    repository: bitnami/kubectl
    tag: 1.25.8
```